pilot/pkg/networking/deprecated/config.go

```diff
@@ -0,0 +1,836 @@
+// Copyright 2018 Istio Authors
+//
```

Comment:
Note. There is no change to this file, except for changing buildListeners to BuildListeners, and removing any v1alpha3 code (gateway/etc.)